src/main/java/tools/jackson/dataformat/xml/deser/FromXmlParser.java

```diff
@@ -35,10 +35,10 @@ public class FromXmlParser
     implements ElementWrappable
 {
     /**
-     * The default name placeholder for XML text segments is empty
-     * String ("").
+     * The default name placeholder for XML text segments is &lt;xml:text&gt;
+     * @since 3.0 Is now &lt;xml:text&gt; - was empty String ("") before.
      */
-    public final static String DEFAULT_UNNAMED_TEXT_PROPERTY = "";
+    public final static String DEFAULT_TEXT_PROPERTY = "<xml:text>";
```

Comment:
Let's rename as "DEFAULT_XML_TEXT_PROPERTY" -- will do.

Reply:
Oh. Actually, n/m -- this is at lower level, where `@XmlText` is not considered. Rather it is for "anonymous" text segments. Will slightly reword description but will not change constant name.